packages/orbit-components/src/Card/types.d.ts

```diff
@@ -6,10 +6,13 @@ import type * as React from "react";
 import type * as Common from "../common/types";
 import type { As } from "../Heading/types";
 
+// TODO: remove spaceAfter in the next major version
+/** spaceAfter is deprecated, use margin instead */
 export interface Props extends Common.Globals, Common.SpaceAfter {
   readonly children?: React.ReactNode;
   readonly title?: React.ReactNode;
   readonly titleAs?: As;
+  readonly margin?: string | Common.ObjectProperty;
```

Comment:
Isn't it safer if we define the type as
```suggestion
  readonly margin?: React.CSSProperties["margin"] | Common.ObjectProperty;
```

This applies for all, ofc. WDYT?

Reply:
Hmm, always thought it was mostly for style prop, but it works  ok, except that the only difference is that it will allow string and number.

Reply:
Yeah, it doesn't offer that much validation in the end (eg: it could validate the value passed is valid to define a margin). But it at least complies with the system, if they ever decide to apply that kind of validation 😄